<launch>
  <!-- <include file="$(find lslidar_c16_decoder)/launch/lslidar_c16.launch" />
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="/camera" />
    <param name="camera_info_url" value="file://$(find calibration)/params/calibrated_camera_1226.yaml" />
    <param name="io_method" value="mmap"/>
  </node> -->
  <node pkg="calibration" type="lidar_cam" name="lidar_cam" output="screen">
    <rosparam file="$(find calibration)/params/1226_calibration_2.yaml" command="load" />
  </node>
</launch>